### PR TITLE
Add 10% for storage metadata to the total required space (#1578395)

### PR DESCRIPTION
**Pull request by poncovka**
In the Install Options dialog, we should add 10% of the disk space for
storage metadata to the total required space. Otherwise, the space
requirements might be confusing.

Resolves: rhbz#1578395

### CODE REVIEW

**Comment by poncovka** [comment]
Now we will show a required size that is higher than the available space.

screenshot_vpn_test_02_2018-05-16_16 03 30

File: pyanaconda/ui/gui/spokes/storage.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def _get_sw_needs_text(self, required_space, auto_swap):
                          "space, including <b>%(software)s</b> for software and "
                          "<b>%(swap)s</b> for swap space.")
                        % {"product": escape_markup(productName),
-                          "total": escape_markup(str(required_space + auto_swap)),
+                          "total": escape_markup(str(required_space + auto_swap + 0.1 * disk_free)),
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/storage.py]: Why 0.1 constant? Could you please make the reason more obvious? Ideally make it `METADATA_SIZE` constant.

File: pyanaconda/ui/gui/spokes/storage.py [hunk 2]
```diff
@@ -974,10 +974,10 @@ def _check_space_and_get_dialog(self, disks):
         log.debug("disk free: %s  fs free: %s  sw needs: %s  auto swap: %s",
                   disk_free, fs_free, required_space, auto_swap)
 
-        # compare only to 90% of disk space because fs takes some space for a metadata
-        if (disk_free * 0.9) >= required_space + auto_swap:
+        # We need enough space for the software, the swap and 10% of the disk space for metadata.
+        if disk_free >= required_space + auto_swap + 0.1 * disk_free:
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/storage.py]: OK we are taking swap into account but what about `/boot`, `/home` and similar?

File: pyanaconda/ui/gui/spokes/storage.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/ui/gui/spokes/storage.py]: Added a constant.

File: pyanaconda/ui/gui/spokes/storage.py (hunk 2 above)
Comment (poncovka) [reply pyanaconda/ui/gui/spokes/storage.py]: Added a comment.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**